packages/@sanity/desk-tool/src/panes/documentPane/timeline/userAvatarStack.tsx

```diff
@@ -2,8 +2,8 @@
 ///<reference types="@sanity/types/parts" />
```

Comment:
This TS comment is no longer needed since all parts have been removed 🎉

Reply:
Removed!